pkg/rsyslog/Dockerfile

```diff
@@ -1,10 +1,12 @@
+FROM linuxkit/memlogd:v0.5-amd64 as memlogd
 FROM alpine:3.10.3
 WORKDIR /
 RUN apk add --no-cache \
             rsyslog=8.1904.0-r1 \
             rsyslog-mmjsonparse=8.1904.0-r1
 COPY rsyslog.conf /etc/rsyslog.conf
 COPY init.sh /init.sh
+COPY --from=memlogd /usr/bin/logread /usr/bin/logread
```

Comment:
The problem with this statement is that, in general, logread is not guaranteed to work in the new container.

Reply:
If we separate and run logread from a different container, would it run without any issues?